sdk/typescript/src/types/transactions.ts

```diff
@@ -239,7 +252,7 @@ export const SuiExecuteTransactionResponse = union([
   }),
   object({
     certificate: optional(CertifiedTransaction),
-    effects: SuiCertifiedTransactionEffects,
+    effects: SuiFinalizedEffects,
     confirmed_local_execution: boolean(),
   }),
```

Comment:
Changing this will make the TS-SDK incompatible with TestNet and DevNet. Instead of changing this this we should  mark the existing one as deprecated like line 245, and add a new one to the union

```
object({
    certificate: optional(CertifiedTransaction),
    effects: SuiFinalizedEffects,
    confirmed_local_execution: boolean(),
  })
```

Reply:
No network is using this version of definition yet. DevNet is still using the one in 245. This version is recently added by @longbowlu 2 days ago, which hasn't shipped to any production network yet.